flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -220,10 +222,10 @@ lift_coef_pitch_rate = -57.116 ; The change in lift per change in pitch rate
 lift_coef_daoa = 0 ; lift per change in angle of attack rate
 lift_coef_delta_elevator = -1.652 ; The change in lift per change in elevator deflection
 lift_coef_horizontal_incidence = 0 ; The change in lift per change in horizontal incidence angle
-lift_coef_flaps = 1.9 ; Change in lift due to flaps
+lift_coef_flaps = 1.715 ; Change in lift due to flaps
 lift_coef_spoilers = -0.25 ; Change in lift due to spoilers
 drag_coef_zero_lift = 0.0197 ; The zero lift drag polar
-drag_coef_flaps = 0.1128
+drag_coef_flaps = 0.0778
```

Comment:
are these changes intentional?

Reply:
yes, of course

Reply:
To give a little more context, those values are essentially the full flaps/slats lift and drag coefficients. (Notice the lift/drag factors for full flaps/slats are 1.00). Any time flaps/slats have to be redone, those values will change. The other configurations then have factors applied to these increments to determine their lift/drag coefficients.